ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml

```diff
@@ -0,0 +1,41 @@
+build_root:
+  image_stream_tag:
+    name: redhat-appstudio-e2e-test-runner
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: Dockerfile
+  to: redhat-appstudio-integration-service-image
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: integration-service-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.11"
```

Comment:
switching to 4.12 here would probably make sense as well (although I think we're using 4.11 in other jobs - but it should be updated)

Reply:
Updated. Thanks